### PR TITLE
Added loadHtml

### PR DESCRIPTION
**Pull request by sterlingwes**
Enables loading plain HTML fragments from external sources (ie: public/ static directory) for situations where conditional loading is more appropriate than bundling for all.

HTML is instantiated as a Meteor Template and added to the Template namespace to allow for binding events, etc.

Considering ways to allow use of Handlebar expressions which requires a form of precompiled Handlebar templates for Meteor's Handlebar runtime implementation.

### CODE REVIEW

File: lib/external-file-loader.js [hunk 1]
```diff
@@ -40,6 +40,27 @@
 
 		self.loadedUrls[url] = true;
 	}
+	
+	Loader.prototype.loadHtml = function(url,template_name) {
+		if(Template[template_name])
```
Comment (davidd8) [inline lib/external-file-loader.js]: Please add braces (i.e. "{" and "}") to stay consistent with the rest of the file.

File: lib/external-file-loader.js [hunk 2]
```diff
@@ -40,6 +40,27 @@
...
+			return Template[template_name];
+	
+		var raw = '';
+		if(typeof url !== "string")
+			raw = url();
```
Comment (davidd8) [inline lib/external-file-loader.js]: I don't quite understand this case. Are you expecting calls to loadHtml with objects and not actual urls?

File: README.md [hunk 3]
```diff
@@ -33,12 +33,48 @@ Template.billing.created = function() {
...
+			}
+		});
+		
+		return tpl();
+	});
+	
+	$('#inserthere').html(fragment);
+});
+```
+
```
Comment (davidd8) [inline README.md:75]: Could you show the example htmlFragment.html to make it a bit more clear?

**Comment by davidd8** [comment]
Thanks for the pull request, loading html fragments should be a great addition to the package! Once you address the comments we can go ahead and merge (looks good for the most part).

File: lib/external-file-loader.js (hunk 2 above)
Comment (sterlingwes) [inline lib/external-file-loader.js]: Not sure how to load a static html file from packages.js during test run, so this was a roundabout way of stubbing jQuery.ajax. I've got a more elegant solution on the way.

**Comment by mikey0000** [comment]
Hey, Love the html feature! can you update the atmosphere package please?

**Comment by davidd8** [comment]
Oh yes, forgot about that, should be pushed now!

**Pull request merged**